modules/apps/export-import/export-import-report-service/src/main/java/com/liferay/exportimport/report/internal/empty/model/EmptyModelManagerImpl.java

```diff
@@ -78,6 +79,26 @@ public <T, E extends Exception> T getOrAddEmptyModel(
 			UnsafeBiFunction<String, Long, T, E>
 				getByExternalReferenceCodeUnsafeBiFunction,
 			long groupId)
+		throws E, PortalException {
```

Comment:
@gabrielwas do we need to declare that this method will throw a `PortalException`? 

I thought that `E extends Exception` would be enough to handle the exception thrown by `getGroup` method.